src/templates/base.html

```diff
@@ -82,9 +82,9 @@
       </div>
       <div class="md:w-2/3 content text-xs">
         <p id="appendix" class="font-black text-base">Appendix</p>
-        <p>PowerVoting uses the closeness of elections and the constituency size (e.g., representative,
+        <p>PowerVoting uses the closeness of elections and the constituency size (e.g. voters per representative,
           senator, or elector) at the federal level to find the relative “impact” of a vote for the addresses you look
-          up.</p>
+          up. The Voting Impact Score is the percentage of Vote Slices that could remain unchanged forç the results of an election to still flip. The higher this number, the less movement necessary to flip the election and the greater the impact of a vote.  </p>
```

Comment:
Think there are some formatting issues on this line. `forç` and the two extra spaces at the end looks like an error.